pkg/reconciler/eventlistener/resources/container.go

```diff
@@ -26,6 +27,12 @@ import (
 
 type ContainerOption func(*corev1.Container)
 
+var (
+	MetricsConfig = `{"Domain":"` + TriggersMetricsDomain + `","Component":"eventlistener","ConfigMap":{}}`
+	zapConfig     = `{"level": "info","development": false,"sampling": {"initial": 100,"thereafter": 100},"outputPaths": ["stdout"],"errorOutputPaths": ["stderr"],"encoding": "json","encoderConfig": {"timeKey": "ts","levelKey": "level","nameKey": "logger","callerKey": "caller","messageKey": "msg","stacktraceKey": "stacktrace","lineEnding": "","levelEncoder": "","timeEncoder": "iso8601","durationEncoder": "","callerEncoder": ""}}`
```

Comment:
pretty print the json for a human to be able to edit?

Reply:
This is moved from elsewhere, so I intentionally left it alone.